src/cryptography/x509.py

```diff
@@ -237,6 +237,57 @@ def __repr__(self):
                 "path_length={0.path_length})>").format(self)
 
 
+class KeyUsage(object):
+    def __init__(self, usages):
```

Comment:
Gosh, my instinct is towards a bitfield or something. Not a fan of the list of bools with magic indices.